addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java

```diff
@@ -56,6 +56,7 @@
     private static final String FIELD_CONFIDENCE = "confidence:";
     private static final String FIELD_VERSION = "version:";
     private static final int SIZE = 16;
+    private static final String ERROR_INVALID_FIELD = "Invalid field syntax {} : {}";
```

Comment:
I'd suggest catch the exception in `parseConfidence` and return a default value. (In any case we should fail the test on parsing errors.)